modules/openapi-generator/src/main/resources/java-micronaut/client/api.mustache

```diff
@@ -45,7 +45,10 @@ import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 	{{{.}}}
 {{/additionalClientTypeAnnotations}}
 {{>common/generatedAnnotation}}
-@Client("${{openbrace}}{{{applicationName}}}-base-path{{closebrace}}")
+@Client(
+  {{#configureClientId}}id = "{{clientId}}",{{/configureClientId}}
+  path = "${{openbrace}}{{{applicationName}}}{{basePathSeparator}}base-path{{closebrace}}"
+)
```

Comment:
Could you change it to 
```txt
@Client({{#configureClientId}}
  id = "{{clientId}}",{{/configureClientId}}
  path = "${{openbrace}}{{{applicationName}}}{{basePathSeparator}}base-path{{closebrace}}"
)
```
so that there wouldn't be an extra line in the generated code?

Reply:
@thrykol Is there anyway of omitting the path? I'd like to specify a client id but keep the path in my `application.yml`.

Reply:
What if you don't want to make use of the generated yml? I only want to generate the declarative client and specify the client id.

Reply:
Hmm, yes I suppose it is better to set the URL with `micronaut.http.services.{id}.url` if the `@Client(id=)` is used. This supports all kinds of use cases.

Reply:
What would be the best way to achieve this @andriy-dmytruk? Adding an `includePath` config option which defaults to `true`?

Reply:
Probably removing the path all along and simply make it configurable in the config by the client id would be the best.